packages/mobx/src/api/tojs.ts

```diff
@@ -63,6 +63,10 @@ function toJSHelper(source, __alreadySeen: Map<any, any>) {
 
 /**
```

Comment:
Thank you for the improvement, could you please replace whole comment with:
```
/**
 * Recursively converts an observable to it's non-observable native counterpart.
 * It does NOT recurse into non-observables, these are left as they are, even if they contain observables.
 * Computed and other non-enumerable properties are completely ignored.
 * Complex scenarios require custom solution, eg implementing `toJSON` or using `serializr` lib.
 */
```
"deep copy but not really" bit was just confusing
Observables and non-observables can be mixed even without classes, no need to be specific about these.

Reply:
Agreed. It makes it much clearer, thanks for the suggestion 👍